### PR TITLE
Enable all tests in the TITUS_INTEGRATION_TEST phase and fix the race condition in ZookeeperMasterMonitorTest

### PR DESCRIPTION
**Pull request by tbak**
@fabiokung Contains your changes from https://github.com/Netflix/titus-control-plane/pull/482 + bug fix

### CODE REVIEW

File: buildViaTravis.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # This script will build the project.
 
 if [ "$TITUS_INTEGRATION_TEST" == "true" ]; then
-  echo -e "Running integration tests (PR $TRAVIS_PULL_REQUEST) => Branch [$TRAVIS_BRANCH]"
-  ./gradlew integrationTest --stacktrace
+ echo -e "Running integration tests (PR=$TRAVIS_PULL_REQUEST) => Branch [$TRAVIS_BRANCH]"
```
Comment (fabiokung) [inline buildViaTravis.sh]: fix identation (1 space)

**Comment by fabiokung** [comment]
LTGM, but it seems one of the tests is still flaky.

File: titus-ext/zookeeper/src/test/java/com/netflix/titus/ext/zookeeper/supervisor/ZookeeperMasterMonitorTest.java [hunk 2]
```diff
@@ -146,7 +148,12 @@ public void testLocalMasterInstanceUpdates() throws Exception {
     private void expectMasters(ExtTestSubscriber<List<MasterInstance>> mastersSubscriber, MasterInstance... masters) throws Exception {
         List<MasterInstance> last = null;
         for (List<MasterInstance> next; (next = mastersSubscriber.takeNext()) != null; ) {
```
Comment (fabiokung) [inline titus-ext/zookeeper/src/test/java/com/netflix/titus/ext/zookeeper/supervisor/ZookeeperMasterMonitorTest.java]: do we need a `.timeout()` here on the `takeNext()`?

File: buildViaTravis.sh (hunk 1 above)
Comment (tbak) [reply buildViaTravis.sh]: Fixed

File: titus-ext/zookeeper/src/test/java/com/netflix/titus/ext/zookeeper/supervisor/ZookeeperMasterMonitorTest.java (hunk 2 above)
Comment (tbak) [reply titus-ext/zookeeper/src/test/java/com/netflix/titus/ext/zookeeper/supervisor/ZookeeperMasterMonitorTest.java]: I had to rewrite this code. Please, check my next PR.
There is different ZK behavior in Travis in when running on my Mac.

**Pull request merged**